setup.py

```diff
@@ -26,6 +26,20 @@
 # http://blog.ionelmc.ro/2014/05/25/python-packaging/
 
 _PATH_ROOT = os.path.dirname(__file__)
+_PATH_REQUIRE = os.path.join(_PATH_ROOT, "requirements")
+
+extras = {
+    "docs": setup_tools._load_requirements(path_dir=_PATH_REQUIRE, file_name="docs.txt"),
+    "test": setup_tools._load_requirements(path_dir=_PATH_REQUIRE, file_name="test.txt"),
+    "image": setup_tools._load_requirements(path_dir=_PATH_REQUIRE, file_name="image.txt"),
+    "video": setup_tools._load_requirements(path_dir=_PATH_REQUIRE, file_name="video.txt"),
+    "tabular": setup_tools._load_requirements(path_dir=_PATH_REQUIRE, file_name="tabular.txt"),
+    "text": setup_tools._load_requirements(path_dir=_PATH_REQUIRE, file_name="text.txt"),
+}
+
+extras["vision"] = extras["image"] + extras["video"]
+extras["dev"] = extras["vision"] + extras["tabular"] + extras["text"] + extras["test"]
+extras["all"] = extras["dev"] + extras["docs"]
```

Comment:
Could "dev" and "all" be switched? because, for someone who wants the full feature, I think it is more natural to install it with `pip install lightning-flash[all]`.
```suggestion
extras["all"] = extras["vision"] + extras["tabular"] + extras["text"]
extras["dev"] = extras["all"] + extras["docs"] + extras["test"]
```

Reply:
I agree